### ISSUE TITLE
Mistress Sassz'ine, SayCountdown() and Shadow Priests

### ISSUE THREAD

**Issue by Zireael-N** [BigWigsMods/BigWigs#340]
This encounter is probably the first one where Voidform is giving me a headache. My guild, and, I assume, many others, are using a strategy where a WA string assigns people to soak a certain mark's Hydra Shot. The thing is, it is very hard to see a raid mark behind a chat bubble. Normally, it is not a problem, because the chat bubble contains the mark. But Shadow Priests' chat bubbles do more harm than good, especially so if you have more than one in your raid group. So I was wondering if you think it's a good idea to just silence a message coming from SayCountdown() if Voidform is present, for this boss at least.

**Comment by funkydude** [comment]
Have you considered rerolling to a better class?

**Comment by Zireael-N** [comment]
Man, I am not playing Shadow myself. :/

I can make a patch myself, but I want to know if it's a welcome change in the upstream.

**Comment by Justw8** [comment]
I rather have some say warning displayed than none as an indicator something is happening even if you cannot see the icon (since they still have a raid icon above their head). But feel free to share it with your guild's shadow priests if that is what you prefer!

**Comment by Zireael-N** [comment]
> none as an indicator something is happening
Well, my patch leaves the initial say message as is: https://github.com/BigWigsMods/BigWigs/pull/342/files#diff-8d397082e7b6a98d80ad1eff2f35a628R186

> (since they still have a raid icon above their head)
They do but chat bubbles overlap with those and it makes it harder to see them.

> feel free to share it with your guild's shadow priests
Unfortunately, that would require me to make a patched version each time a new version of BigWigs releases. I worked around this by making a WA string that makes SayCountdown() point to a modified function because importing it once is just easier. I also shared it on wago.io, and it currently has 900 views, so I'd say there are quite a few people who prefer this behaviour. I'm not sure if links to external resources are welcome here, but if you're curious, that WA is easily google-able.
